src/loaders/routes.ts

```diff
@@ -2,12 +2,14 @@ import { Router } from 'express';
 
 import roomsApi from '../modules/rooms/RoomApi';
 import usersApi from '../modules/users/UserApi';
+import room_membersApi from "../modules/room_members/Room_membersApi";
 
 
 export default (): Router => {
   const app = Router();
 
   usersApi(app);
   roomsApi(app);
+  room_membersApi(app);
```

Comment:
NUNCA PODES USAR `_` nos nomes... a única excepção é a convenção dos `this._<varname>` para as variáveis privadas, de resto, tudo é camel case (e claro, na referência aos campos da BD)

Reply:
Não vou comentar em todo o lado... nomes de ficheiros, nomes de pastas, e nomes de variáveis, tens de os mudar todos!